packages/core/src/components/autosuggest/readme.md

```diff
@@ -0,0 +1,25 @@
+# `<dso-autosuggest>`
+
+Todo @ben
```

Comment:
@benheldoorn Wil jij hier nog een beetje developer documentatie plaatsen?

Ik had gehoopt dat interfaces automatisch zouden worden meegedocumenteerd, maar dat is niet het geval. Bij Ionic plakken ze de interface er ook los bij, dus ik dat heb ik er vast bijgezet.

Reply:
Documentatie is aangevuld.

Reply:
Looking sharp